src/gimelstudio/core/node/node.py

```diff
@@ -238,7 +238,7 @@ def EvalMutedNode(self, eval_info):
             "image": render_image
         }
 
-    def RenderGLSL(self, path, props, image=Image(200, 200), image2=None):
+    def RenderGLSL(self, path, props, image=Image((200, 200)), image2=None):
```

Comment:
I think we should merge this PR right way just to get this fix. Without it, the ``master`` branch code won't run as-is.